React/game/src/app/screens/Login/components/Login/layout.js

```diff
@@ -17,9 +19,16 @@ function LoginLayout(props) {
         <button className={styles.button} type="submit">
           Submit
         </button>
+        {!isLogin && <span>{msgError}</span>}
```

Comment:
again... if the user is not logged in, there's no explicit reason to display an error. Rely on `msgError` to display if there are any errors.

Reply:
👍